CorsixTH/Lua/room.lua

```diff
@@ -202,20 +202,6 @@ function Room:dealtWithPatient(patient)
   if self.dealt_patient_callback then
     self.dealt_patient_callback(self.waiting_staff_member)
   end
-  -- The staff member(s) might be needed somewhere else.
-  self:findWorkForStaff()
-end
-
---! Checks if the room still needs the staff in it and otherwise
--- sends them away if they're needed somewhere else.
-function Room:findWorkForStaff()
-  -- If the staff member is idle we can send him/her somewhere else
-  for humanoid in pairs(self.humanoids) do
-    -- Don't check handymen
-    if class.is(humanoid, Staff) and humanoid.humanoid_class ~= "Handyman" and humanoid:isIdle() then
-      self.world.dispatcher:answerCall(humanoid)
-    end
-  end
 end
```

Comment:
I still like he idea of having an immediate find work at the end of the consult. Is there a specific consequence if we left this in with the new code?

I suppose the worst of it is that the staff member takes a day to find new work.

Reply:
On reflection I think I prefer the balance you have chosen, so happy to discuss my above comment.

Reply:
I did toss up leaving it in as well, hence leaving it more as a proposal. The original has decisions and logic processed on part day ticks. I assume so some logic doesn't run every tick but also some more frequently than once per day. This probably sits in one of those in the original.

Reply:
Actually doesn't this function only get called from GP Office?

Reply:
As per above. Only GP Office used this function. Maybe that was part of the problem too.